packages/teams-js/src/public/authentication.ts

```diff
@@ -137,6 +137,7 @@ export namespace authentication {
         GlobalVars.hostClientType === HostClientType.android ||
         GlobalVars.hostClientType === HostClientType.ios ||
         GlobalVars.hostClientType === HostClientType.ipados ||
+        GlobalVars.hostClientType === HostClientType.macos ||
```

Comment:
It would be good to verify that this change isn't going to change the behavior of authentication on Mac Teams clients that currently exist in a way that is unexpected or undesirable.

Reply:
Just curious, did we hear anything back from Teams mac on this ?

Reply:
We discussed this matter in last Tuesday's meeting. Currently, Teams Mac clients utilize the "desktop" hostClientType. I intend to continue this discussion and reach a conclusion in tomorrow's meeting.

Reply:
Conclusion from today's meeting: The existing mac Teams clients will continue using the "Desktop" hostClientType to ensure backward compatibility. Conversely, the new Mac Teams clients, which will incorporate the MacHubSDK, will use the new "macos" hostClientType. This differentiation will also aid us in distinguishing between Mac clients utilizing the webHubSDK and those using the macHubSDK. This would make it more scalable, provide better control over individual sdk usage in the future, and ensure backward compatibility.